src/main/webapp/index.html

```diff
@@ -206,13 +206,37 @@ <h3 hidden class="panel__section-heading u-remove-top-margin assign-settings">Se
         </div>
         <div id="assign"></div>
       </div>
-      <div class="panel">Plan
-        <p> Your unread emails contain <b id="word-count"></b> words
-        <p>Average reply: <b id="average-reply"></b> words
-        <p>Time needed: <b id="time-needed"></b> minutes
-        <p id="message-event">
-        <p id="free-interval">
+      <div class="panel u-border-plan">
+        <div class="panel__header">
+          <div class="panel__content-entry u-remove-top-margin">
+            <a class="panel__content-entry-link" target="_blank" href="https://calendar.google.com/calendar/b/0/r">
+              <img class="panel__logo" alt="plan-logo" src="assets/plan_tick.png">
```

Comment:
Do you need to add any new attributions for these assets at the bottom of the page?

Reply:
No, I am using the same links.